### DISCUSSION TITLE
< Reclaim GitHub Student developer pack >

### DISCUSSION THREAD

**Discussion by Rishabh-391** [community/community#63088] [GitHub Education]
### Select Topic Area

Question

### Body

My GitHub Student developer pack is going to expire in few days and I'm still a student how may I reclaim the pack?

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Rishabh-391, we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!
